data-es6.js

```diff
@@ -9375,23 +9375,53 @@ exports.tests = [
       },
     },
     {
-      name: 'JSON.stringify ignores symbols',
+      name: 'JSON.stringify ignores symbol primitives',
       exec: function() {/*
-        var object = {foo: Symbol()};
+        var object = { foo: Symbol() };
         object[Symbol()] = 1;
         var array = [Symbol()];
         return JSON.stringify(object) === '{}' && JSON.stringify(array) === '[null]' && JSON.stringify(Symbol()) === undefined;
       */},
       res: {
         babel: true,
         typescript: typescript.corejs,
-		es6shim: true,
+        es6shim: true,
         firefox36: true,
+        firefox40: true,
```

Comment:
The goal was to mark a version that's explicitly tested. Browsers do occasionally (both unintentionally and intentionally) change their support of a behavior, so it's useful to have explicitly checked.